README.md

```diff
@@ -2,8 +2,9 @@
 
 [![Stable](https://img.shields.io/badge/docs-stable-blue.svg)](https://matrixfunctions.github.io/GraphMatFun.jl/stable)
 [![Dev](https://img.shields.io/badge/docs-dev-blue.svg)](https://matrixfunctions.github.io/GraphMatFun.jl/dev)
-[![Build Status](https://travis-ci.com/matrixfunctions/GraphMatFun.jl.svg?token=1D6L5XDzryqshBZ5ijUH&branch=main)](https://travis-ci.com/matrixfunctions/GraphMatFun.jl)
+[![Tests](https://img.shields.io/github/workflow/status/matrixfunctions/GraphMatFun.jl/ContinuousIntegration?label=tests)](https://github.com/matrixfunctions/GraphMatFun.jl/actions/workflows/ContinuousIntegration.yml)
 [![codecov](https://codecov.io/gh/matrixfunctions/GraphMatFun.jl/branch/main/graph/badge.svg?token=ZTKNBNMDEZ)](https://codecov.io/gh/matrixfunctions/GraphMatFun.jl)
```

Comment:
What is the role of the "token" now? The discourse example https://raw.githubusercontent.com/invenia/PkgTemplates.jl/master/README.md suggests it would be `[![Codecov](https://codecov.io/gh/matrixfunctions/GraphMatFun.jl/branch/master/graph/badge.svg)](https://codecov.io/gh/matrixfunctions/GraphMatFun.jl)`

Reply:
For our repository the query string is necessary, and in fact the badge doesn't work without it: compare

[https://codecov.io/gh/matrixfunctions/GraphMatFun.jl/branch/master/graph/badge.svg](https://codecov.io/gh/matrixfunctions/GraphMatFun.jl/branch/master/graph/badge.svg)

with 

[https://codecov.io/gh/matrixfunctions/GraphMatFun.jl/branch/main/graph/badge.svg?token=ZTKNBNMDEZ](https://codecov.io/gh/matrixfunctions/GraphMatFun.jl/branch/main/graph/badge.svg?token=ZTKNBNMDEZ).

The markdown snippet is the one recommended on the Codecov settings for our repository:

[https://app.codecov.io/gh/matrixfunctions/GraphMatFun.jl/settings/badge](https://app.codecov.io/gh/matrixfunctions/GraphMatFun.jl/settings/badge)